app-office/drawio-desktop/drawio-desktop-22.1.2.ebuild

```
@@ -0,0 +1,137 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="draw.io"
+MY_NODE_N="node-modules"
+MY_NODE_D="node_modules"
+DRAWIO_V="22.1.4"
+
+inherit desktop xdg
+
+DESCRIPTION="draw.io diagramming and whiteboarding desktop app"
+HOMEPAGE="https://www.drawio.com/"
+
+SRC_URI="
+	https://github.com/jgraph/drawio-desktop/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/jgraph/drawio/archive/refs/tags/v${DRAWIO_V}.tar.gz
```

```suggestion
	https://github.com/jgraph/drawio/archive/refs/tags/v${DRAWIO_V}.tar.gz -> ${PN}-${DRAWIO_V}.tar.gz
```

v22.1.4.tar.gz is very generic